lib/eslint/legacy-eslint.js

```diff
@@ -171,6 +171,7 @@ function processOptions({
     extensions = null, // ← should be null by default because if it's an array then it suppresses RFC20 feature.
     fix = false,
     fixTypes = null, // ← should be null by default because if it's an array then it suppresses rules that don't have the `meta.type` property.
+    flags, /* eslint-disable-line no-unused-vars -- leaving for compatibility with ESLint#hasFlag */
```

Comment:
It looks like this line is not required?

Reply:
It is required because `flags` can be passed into `LegacyESLint`. If I didn't include it here, it would end up in `unknownOptions` and cause an error.

Reply:
Of course! I totally missed re-running the tests to check that.